Algorithms.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+
+// find max
```

Comment:
As a reviewer I can add comments in your PR. For example here I think defining getMax and getMin is redundant since they just return max and min and one could use max and min instead of getMax and getMin